{"uuid": "b2e3fd6d-fd15-4c78-bce9-167055aa15bf", "children": ["62caf3c1-1ce7-44a7-83f2-b73070ce0f3a"], "befores": [{"name": "_xunit_setup_class_fixture_TestCreateCourier", "status": "broken", "statusDetails": {"message": "AttributeError: 'dict' object has no attribute 'append'\n", "trace": "  File \"/Users/almihalev/yandex/practicum_qa_python_sprints/Sprint_7_qa_python/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/almihalev/yandex/practicum_qa_python_sprints/Sprint_7_qa_python/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/almihalev/yandex/practicum_qa_python_sprints/Sprint_7_qa_python/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/Users/almihalev/yandex/practicum_qa_python_sprints/Sprint_7_qa_python/venv/lib/python3.11/site-packages/_pytest/python.py\", line 856, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"/Users/almihalev/yandex/practicum_qa_python_sprints/Sprint_7_qa_python/venv/lib/python3.11/site-packages/_pytest/python.py\", line 777, in _call_with_optional_argument\n    func(arg)\n  File \"/Users/almihalev/yandex/practicum_qa_python_sprints/Sprint_7_qa_python/tests/test_list_of_orders.py\", line 15, in setup_class\n    cls.data.append(order)\n    ^^^^^^^^^^^^^^^\n"}, "start": 1701267706881, "stop": 1701267707046}], "start": 1701267706881, "stop": 1701267707054}